README.md

```diff
@@ -219,6 +220,7 @@ Sent from the rover to inform Mission Control of the rover's current position in
 - `posX` - refers to the X position of the rover in world reference frame
 - `posY` - refers to the Y position of the rover in world reference frame
 - `posZ` - refers to the Z position of the rover in world reference frame 
+- `recency` - refers to the difference in time between when the message was sent and received
```

Comment:
This description isn't quite right, that would be something closer to latency. Recency measures the amount of elapsed time between the value being measured and being sent from the rover.